packages/expo-notifications/README.md

```diff
@@ -1,5 +1,115 @@
 # expo-notifications
 
-`expo-notifications` provides some great stuff.
+## Instalation 
```

Comment:
I am not sure if that is clear, which sections describes changes that are required for `expo-notifications` to be used and which describes only optional configuration.
Personally (but that is only an opinion) I prefer readmes, which starts with minimal required configuration, which I don't even need to understand fully to apply, and then explains each feature part by part, for both platforms.

Reply:
I really like the changes you've made.

I have only one general suggestion as for sections and their naming. I would suggest something more like:
#Installation
#Basic configuration
#Local Notifications
#Remote notifications
##With expo servers
##With custom server
#Advanced
##Launching custom activity
#Anything else? Like reference?

If we believe that custom activity is very rare scenario, I would pass it down, because it will only darken the image for majority of users.
What do you think of that?